lib/ansible/module_utils/redfish_utils.py

```diff
@@ -862,7 +903,16 @@ def get_nic_inventory(self, resource_type):
         result["entries"] = nic_results
         return result
 
-    def get_psu_inventory(self):
+    def get_multi_nic_inventory(self, resource_type):
```

Comment:
I think you should be able to use the aggregate() method here if you modify aggregate() to add an optional resource_type param.

Reply:
Yeah I could do that, but isn't it break other "multi" like methods ? :)

Reply:
One approach would be to add a `**kwargs` param to aggregate() as well as to each of the "single" methods that are called by aggregate(). But that seems like overkill for this one get_multi_nic_inventory() special case.

I'm thinking now it would be better to address this get_multi_nic_inventory() special case via a new issue and PR. (It can be changed so that it no longer requires the extra `resource_type` param.)

So I'm OK with leaving get_multi_nic_inventory() as-is in this PR.